lib/python/test/KHI_growthRate/MainTest.py

```diff
@@ -0,0 +1,38 @@
+import sys
+from time import *
+
+lt = localtime()
+date = strftime("date: %d.%m.%Y", lt)
+timeOfDay = strftime("time: %H:%M:%S", lt)
+
+try:
+	sys.path.append(sys.argv[1])
```

Comment:
Why is this required? Does python has no way to get the file location of the executed file?